cli/man/splinter-circuit-show.1.md

```diff
@@ -76,13 +76,14 @@ The information displayed below will appear the same on all proposed member node
 If all member nodes vote to accept the circuit, the `splinter-circuit-show`
 command will display the same information, without the `Vote` as all nodes would
 have accepted the proposal. If any of the member nodes vote to reject the circuit,
-the proposal will not be viewable by any nodes.
+the proposal will be removed and is not be viewable by any nodes.
```

Comment:
" not be viewable by any nodes" -> " not viewable by any nodes"

Reply:
Updated